src/main/java/service/impl/ReaderServiceImpl.java

```diff
@@ -0,0 +1,33 @@
+package service.impl;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+import java.util.stream.Collectors;
+import model.Fruit;
+import service.ParserService;
+import service.ReaderService;
+
+public class ReaderServiceImpl implements ReaderService {
+    private static final String TITLE = "type";
+    private ParserService parserService;
+
+    public ReaderServiceImpl(ParserService parserService) {
+        this.parserService = parserService;
+    }
+
+    @Override
+    public List<Fruit> readFromFile(String fileName) {
```

Comment:
Let's refactor this method to return `List<String>` and get rid of dependency on `ParserService` .
This will more matching co Single Responsibility (SOLID) principle and get ability easily extends this class in the future.